src/plugins/workspace/server/saved_objects/workspace_ui_settings_client_wrapper.ts

```diff
@@ -101,8 +101,9 @@ export class WorkspaceUiSettingsClientWrapper {
       /**
        * When updating ui settings within a workspace, it will update the workspace ui settings,
        * the global ui settings will remain unchanged.
+       * When updating default workspace, it will update user level settings.
        */
-      if (type === 'config' && requestWorkspaceId) {
+      if (type === 'config' && requestWorkspaceId && !(DEFAULT_WORKSPACE in attributes)) {
```

Comment:
@Hailong-am should we check the `id` not starts with `CURRENT_USER_PLACEHOLDER` here?

Reply:
Yes, use `id.startsWith(CURRENT_USER_PLACEHOLDER)` for user level setting.

The order of user setting wrapper is 3 that after this workspace setting wrapper which is -2 (lower execute first), so the id has not been replaced to real user name.

Reply:
Updated